pkg/network/ebpf/c/cookie.h

```diff
@@ -1,12 +1,20 @@
 #ifndef __COOKIE_H__
 #define __COOKIE_H__
 
-#include <linux/types.h>
-#include <net/sock.h>
+#include "ktypes.h"
+#include "bpf_helpers.h"
+
+#include "defs.h"
 
 static __always_inline u32 get_sk_cookie(struct sock *sk) {
-    u64 t = bpf_ktime_get_ns();
-    return (u32)((u64)sk ^ t);
+#if defined(COMPILE_RUNTIME) && LINUX_VERSION_CODE >= KERNEL_VERSION(4, 9, 0)
+    return bpf_get_prandom_u32();
```

Comment:
FYI this means CO-RE will never use `bpf_get_prandom_u32`. If you are OK with that, then no change needed.

Reply:
It would mean we'd have to tid CO-RE to version 4.9.0, right?

Reply:
Yeah, I guess we need to wait until we have kernel version available in CO-RE to make this work correctly.